src/dotty/tools/backend/jvm/DottyBackendInterface.scala

```diff
@@ -678,8 +686,24 @@ class DottyBackendInterface()(implicit ctx: Context) extends BackendInterface{
 
     // members
     def primaryConstructor: Symbol = toDenot(sym).primaryConstructor
-    def nestedClasses: List[Symbol] = memberClasses //exitingPhase(currentRun.lambdaliftPhase)(sym.memberClasses)
-    def memberClasses: List[Symbol] = toDenot(sym).info.memberClasses.map(_.symbol).toList
+
+    /** For currently compiled classes: All locally defined classes including local classes.
```

Comment:
My guess is:

```
class A{
  class B
  def foo = {
    class C
  }
}
```
Class `B` is locally defined, class `C` is local.
Though the naming here is indeed confusing.